docs/sources/tempo/configuration/_index.md

```diff
@@ -731,6 +731,11 @@ storage:
             # Example: "endpoint: s3.dualstack.us-east-2.amazonaws.com"
             [endpoint: <string>]
 
+            # The number of list calls to make in parallel to the backend per instance.
+            # Adjustments here will impact the polling time, as well as the number of Go routines.
+            # Default is 3
```

Comment:
I'm rethinking this and perhaps a default of `1` is better here.  Thoughts?

Reply:
I had tested with 20 on gcs and got the best performance, but this was not running with the large number of instances we see in some of our environments.  The good thing is that we have a configuration option to tune.  I'm fine with coming out the gate with 3 and then finding a nice balance as we roll this out, then come back and adjust our defaults if it turns out we were way off.